### DISCUSSION TITLE
How do I configure saleor locally when DEBUG is off?

### DISCUSSION THREAD

**Discussion by g-londhe** [saleor/saleor#9883] [Deployment]
**How do I configure the saleor locally when DEBUG is off?**

I made the following changes,
1) Generated and added secret_key and RSA_key in backend.env

2) Added the following variables to the API service

```
- API_URI=http://localhost:8000/graphql/
- DASHBOARD_URL=http://localhost:9000
- STOREFRONT_URL=http://localhost:3001
- ALLOWED_HOSTS=localhost,api
- ALLOWED_CLIENT_HOSTS=localhost,api
- DEBUG=False
- ENABLE_ACCOUNT_CONFIRMATION_BY_EMAIL=False
```

Outcome:
1) localhost:8000 stopped working, while localhost:8000/graphql is working fine.
2) Unable to see the media uploaded as a path to the media upload is localhost:8000

I have set up a saleor using the steps mentioned here -> https://saleor.io/open-source/

Kindly advise to make it work with docker-compose and DEBUG off.

Saleor version -> 3.2.0

**Comment by manojLondhe** [comment]
I also noticed 2 issues when debug is off
- localhost:8000 stopped working, while localhost:8000/graphql is working fine.
- Unable to see the media uploaded as a path to the media upload is localhost:8000

**Comment by g-londhe** [comment]
Can someone help me with this?

**Reply by karolkielecki** [reply]
Hi @g-londhe, @manojLondhe -- are you using the saleor-platform repo?

**Reply by karolkielecki** [reply]
Also, what would you like to achieve by having DEBUG off in your cases? You should be able to set an environment variable DEBUG with false

**Reply by manojLondhe** [reply]
@karolkielecki Yes, I used saleor-platform repo.

Before moving DEV server on cloud, I wanted to test it locally with debug off, but doesn't seem to work.

We could not find any docs about best practices to deploy (self-host) Saleor in cloud.

**Reply by g-londhe** [reply]
Hi @karolkielecki,
I'm using saleor-platform.

We would like to deploy our app to production and want to make sure that everything works fine before we move to production.
and it looks like it doesn't.

Any documentation or tips would be helpful for us.
